plugin/src/test/java/io/jenkins/plugins/casc/misc/Util.java

```diff
@@ -203,4 +209,40 @@ public static String convertToJson(String yamlString) {
         JSONObject jsonObject=new JSONObject(map);
         return jsonObject.toString();
     }
+
+    /**
+     * Returns the JSON V7 schema for the running jenkins instance.
+     * Example Usage:
+     *      * <pre>{@code
+     *      * Schema jsonSchema = returnSchema();}
+     *      * </pre>
+     * @return
```

Comment:
Please avoid using empty tags, it just leads to Javadoc warnings

Reply:
Got it :)